docs/examples/viz_network_animated.py

```diff
@@ -0,0 +1,259 @@
+
+    files, folder = fetch_viz_wiki_nw()
+    categories_file, edges_file, positions_file = sorted(files.keys())
+
+###############################################################################
+# We read our datasets (mode 1)
+
+if(mode == 1):
+    positions = np.loadtxt(pjoin(folder, positions_file))
+    categories = np.loadtxt(pjoin(folder, categories_file), dtype=str)
+    edges = np.loadtxt(pjoin(folder, edges_file), dtype=np.int)
+    vertices_count = len(positions)
+
+###############################################################################
+# Generate a geographic random network, requires networkx package (mode 0)
+
+if(mode == 0):
+    import networkx as nx
+    vertices_count = 100
+    view_size = 100
+    network = nx.random_geometric_graph(vertices_count, 0.2)
+    positions = view_size*np.random.random((vertices_count, 3))-view_size/2.0
+    categories = np.arange(0, vertices_count)
+    edges = np.array(network.edges())
+    positions = view_size*np.random.random((vertices_count, 3))-view_size/2.0
+
+###############################################################################
+# We attribute a color to each category of our dataset which correspond to our
+# nodes colors.
+
+category2index = {category: i
+                  for i, category in enumerate(np.unique(categories))}
+
+index2category = np.unique(categories)
+
+category_colors = cmap.distinguishable_colormap(nb_colors=len(index2category))
+
+colors = np.array([category_colors[category2index[category]]
+                   for category in categories])
+
+###############################################################################
+# We define our node size
+
+radii = 1 + np.random.rand(len(positions))
+
+###############################################################################
+# Lets create our edges now. They will indicate a citation between two nodes.
+# OF course, the colors of each edges will be an interpolation between the two
+# node that it connects.
+
+edges_colors = []
+for source, target in edges:
+    edges_colors.append(np.array([colors[source], colors[target]]))
+
+edges_colors = np.average(np.array(edges_colors), axis=1)
+
+
+###############################################################################
+# Our data preparation is ready, it is time to visualize them all. We start to
+# build 2 actors that we represent our data : sphere_actor for the nodes and
+# lines_actor for the edges.
+
+sphere_actor = actor.sphere(centers=np.zeros(positions.shape),
+                            colors=colors,
+                            radii=radii*0.5,
+                            theta=8,
+                            phi=8,
+                            )
+
+
+lines_actor = actor.line(np.zeros((len(edges), 2, 3)),
+                         colors=edges_colors, lod=False)
+lines_actor.GetProperty().SetRenderLinesAsTubes(1)
+lines_actor.GetProperty().SetLineWidth(5)
+lines_actor.GetProperty().SetOpacity(1)
```

Comment:
This can be done in one line like:

```python
lines_actor = actor.line(np.zeros((len(edges), 2, 3)), colors=edges_colors, lod=False, linewidth=5, fake_tube=True)
```

Reply:
Nice, good catch @dmreagan!

Reply:
Thanks, but I remember we discussed that their behavior are different. If I'm not mistaken, one uses fury implementation of fake tubes and the other uses vtk implementation. The shaders are different. Anyway I will check this again and make the other proposed changes.

Reply:
Oh they are the same. Thanks. will make the change.